vtkext/private/module/F3DStyle.cxx

```diff
@@ -0,0 +1,46 @@
+#include "F3DStyle.h"
+
+#include <imgui.h>
```

Comment:
nice approach but I see one problem, F3D_MODULE_UI off means this file is not compiled which means you dont have a color for the progress bar in that case. If we consider the bars to be part of the UI that would be ok but im not so sure about that.